frontend/src/components/PlatformSelect/nds.svg

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   width="32"
+   height="32"
```

Comment:
And also remove the width and height properties as in https://github.com/decompme/decomp.me/commit/ca46e243bdc3ecc02bb2f71a9adea33b502345d1